admin_site/system/views.py

```diff
@@ -591,8 +597,8 @@ def sort_by(a, b):
             # cmp deprecated: cmp converted to key function
             script = sorted(self.scripts, key=cmp_to_key(sort_by))[0]
             return HttpResponseRedirect(script.get_absolute_url(
-            site_uid=self.site.uid
-            ))
+                                        site_uid=self.site.uid)
```

Comment:
This formatting is messy. It should be:
```
            return HttpResponseRedirect(
                script.get_absolute_url(site_uid=self.site.uid)
            )
```

Reply:
Enig. Må have overset det. Det er rettet nu. Jeg skubber alle ændringer senere